packages/graph/hash_graph/lib/graph/src/logging/init.rs

```diff
@@ -46,6 +58,41 @@ where
     }
 }
 
+fn configure_opentelemetry_layer(
+    otlp_endpoint: &str,
+) -> OpenTelemetryLayer<Layered<EnvFilter, Registry>, Tracer> {
+    // Allow correlating trace IDs
+    global::set_text_map_propagator(TraceContextPropagator::new());
+    // If we need to set any tokens in the header for the tracing collector, this would be the place
+    // we do so.
+    let map = MetadataMap::new();
+
+    let pipeline = opentelemetry_otlp::new_exporter()
+        .tonic()
+        .with_endpoint(otlp_endpoint)
+        .with_timeout(std::time::Duration::from_secs(3))
+        .with_metadata(map);
+
+    // This can be changed to ratio-based sampling
+    let trace_config = opentelemetry::sdk::trace::config()
+        .with_sampler(Sampler::AlwaysOn)
+        .with_id_generator(RandomIdGenerator::default())
+        .with_max_events_per_span(64)
+        .with_max_attributes_per_span(16)
+        .with_max_events_per_span(16)
```

Comment:
Same as above, does it makes sense to make these constants?

Reply:
These we may want to configure, yes. I think we should do that in the future when we know what kind of limits we want to set. I think it's fine to leave for later (we can always add the clap args + env variables later)

Reply:
Having this configurable somehow would be good, but having a constant for now is probably good enough for now 🙂

Reply:
The configuration seems good enough for us for now. I'd say we remove the configuration here and rely on their environment variables.

Reply:
![image](https://user-images.githubusercontent.com/6988668/208116825-51faf795-2211-4f7d-b52f-43bb7500ebad.png)
Did a one-off test. Seems to be fine :+1: